frontend/src/components/header/default-elements/default-elements.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import UserIcon from '@assets/images/header/default-avatar.png';
+import { AppRoute } from '@common/enums/app/app-route.enum';
+import { ButtonFill } from '@components/common/button-fill/button-fill';
+import { ButtonOutline } from '@components/common/button-outline/button-outline';
+import styles from '@components/header/styles.module.scss';
+import { useMediaQuery, useTheme } from '@mui/material';
+
+const DefaultElements: React.FC = () => {
```

Comment:
Please use a more explicit name. Because `DefaultElement` doesn't have any semantic meaning or context

Reply:
We had two headers in the project and I combined them into one
I think it's a component, so I renamed it now)